src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1393,6 +1393,41 @@ TxQ::getMetrics(OpenView const& view) const
     return result;
 }
 
+std::tuple<XRPAmount, std::uint32_t, std::uint32_t>
+TxQ::getTxRequiredFeeAndSeq(OpenView const& view,
+    std::shared_ptr<STTx const> const& tx) const
+{
+    auto const account = (*tx)[sfAccount];
+
+    std::lock_guard lock(mutex_);
+
+    auto const snapshot = feeMetrics_.getSnapshot();
+    auto const baseFee = calculateBaseFee(view, *tx);
+    auto fee = FeeMetrics::scaleFeeLevel(snapshot, view);
+
+    auto const sle = view.read(keylet::account(account));
+
+    std::uint32_t accountSeq;
+    if (sle)
+        accountSeq = (*sle)[sfSequence];
+    else
+        accountSeq = 0;
```

Comment:
We could replace this with something like:
```
    auto const accountSeq = [&view, &account]() -> std::uint32_t {
        auto const sle = view.read(keylet::account(account));
        if (sle)
            return (*sle)[sfSequence];
        return 0;
    }();
```
That restricts the scope of the `sle` and lets us make `accountSeq` `const`

Reply:
Done